powerline/listers/i3wm.py

```diff
@@ -0,0 +1,80 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+from powerline.lib.dict import updated
+
+
+conn = None
```

Comment:
This should be near `workspace_lister`. Though no, since connection is shared between lister and segment it (not it directly, but function that gets connection) should be in `powerline/bindings/wm/__init__.py`, with `get_connected_xrandr_outputs`: these two functions are long enough.